WcaOnRails/app/models/team.rb

```diff
@@ -6,6 +6,27 @@ class Team < ApplicationRecord
 
   default_scope -> { where(hidden: false) }
 
+  # "Official" teams are teams recognized by Motion "10.2019.0":
+  #  https://www.worldcubeassociation.org/documents/motions/10.2019.0%20-%20Committees%20and%20Teams.pdf
+  # Motions starting with "10.YYYY.N" define these teams:
+  #  https://www.worldcubeassociation.org/documents
+  scope :official, -> {
+    where(
+      id: [
+        Team.wrt.id,
+        Team.wrc.id,
+        Team.wfc.id,
+        Team.wst.id,
+        Team.wqac.id,
+        Team.wct.id,
+        Team.wmt.id,
+        Team.wcat.id,
+        Team.wdc.id,
+        Team.wec.id,
+      ],
```

Comment:
A matter of preference, I'd list the teams and call `.map(&:id)`, whatever you like more =)

Reply:
I took this a step further and pulled this list of teams out into a new `Team.all_official` method to replace our `Team.all_ordered_by_english_name`. I also updated some places to use this new method. This turned out kind of nicely, because the two places where we were looping over all the teams, we were intentionally filtering out the Board. This is no longer necessary because the Board is not an "official team", so it's not in this list.